torch/csrc/jit/codegen/cuda/tensor_view.cpp

```diff
@@ -435,28 +450,77 @@ TensorView* TensorView::cache_before() {
   // Expr* producer_uses =
```

Comment:
Small nit above, not related to this PR. origin_expr is no longer valid because we replaced it, may be safe for future updates if we set it to nulllptr.